server/controllers/article.js

```diff
@@ -211,10 +210,32 @@ class ArticleController {
       }],
       attributes: ['id', 'slug', 'title', 'description', 'body', 'createdAt', 'updatedAt']
     })
-      .then(article => res.status(200).json({
-        article,
-        status: 'success'
-      }))
+      .then((article) => {
+        if (!article) {
+          return res.status(404).json({
+            status: 'error',
+            error: 'Article does not exist'
+          });
+        }
+        // Record users view in the Article View table
+        const articleId = article.id;
+        const { userId } = req;
+        ArticleView.findOne({
+          where: { userId, articleId },
+        })
+          .then((userView) => {
+            if (!userView) {
+              ArticleView.create({ articleId, userId })
+                .then(() => res.status(201).json())
+                .catch(next);
```

Comment:
The name of this method is `getSpecific`, I think it is not really descriptive. The method name should describe what it is getting. Also in lines`228-230` a resource is created which doesn't seem to tally with what I think the method should be doing.

I think you should have a separate method that creates an articleView.

Reply:
The name `getSpecific` was the work of @ozezzy, but I'll fix it now.

Reply:
Will work on that